### PR TITLE
An ebuild for libarrow/libparquet

### PR DESCRIPTION
**Pull request by sumerman**
An ebuild for libarrow/libparquet that are required for Postgres FDW (ebuild is coming soon).

### CODE REVIEW

File: dev-libs/libarrow/libarrow-0.13.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+EAPI=7
```
Comment (maxeaubrey) [inline dev-libs/libarrow/libarrow-0.13.0.ebuild]: Needs a copyright header.

File: dev-libs/libarrow/libarrow-0.13.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+inherit eutils cmake-utils 
+
+ARROW_VSN="0.13.0"
```
Comment (maxeaubrey) [inline dev-libs/libarrow/libarrow-0.13.0.ebuild]: Why use this instead of `${PV}`?

Comment (sumerman) [reply dev-libs/libarrow/libarrow-0.13.0.ebuild]: I didn't know about it at first and then forgot to change it. Good catch! Fixed.

File: dev-libs/libarrow/libarrow-0.13.0.ebuild (hunk 1 above)
Comment (sumerman) [reply dev-libs/libarrow/libarrow-0.13.0.ebuild]: done

**Comment by maxeaubrey** [comment]
The main reason it's failing right now is the +x on the ebuild. If you want to fix up the warnings, there's some whitespace and an extraneous newline at the end too :)

Besides that, i cannot get it to build:

```
CMake Error at thrift_ep-stamp/download-thrift_ep.cmake:157 (message):
  Each download failed!

    error: downloading 'http://archive.apache.org/dist/thrift/0.12.0/thrift-0.12.0.tar.gz' failed
         status_code: 6
         status_string: "Couldn't resolve host name"
         log:
         --- LOG BEGIN ---
         getaddrinfo(3) failed for archive.apache.org:80
```

Downloads cannot happen anywhere outside of src_unpack afaik, but why is emake handling this at all?

**Comment by sumerman** [comment]
Thank you! The error message was quite cryptic enough for me to miss it. It works on inside VM on my machine, but you are right — all those deps should be pulled by explicitly. `emake` does it because it calls into `cmake` and `libarrow` has some magic dependency handling.

**Comment by maxeaubrey** [comment]
Closing in favour of #268.

**Pull request closed without merging**